pychromecast/socket_client.py

```diff
@@ -64,12 +64,11 @@
 CONNECTION_STATUS_FAILED_RESOLVE = "FAILED_RESOLVE"
 # The socket connection was lost and needs to be retried
 CONNECTION_STATUS_LOST = "LOST"
-# Check for select poll method
-SELECT_HAS_POLL = hasattr(select, "poll")
 
 HB_PING_TIME = 10
 HB_PONG_TIME = 10
-POLL_TIME_BLOCKING = 5.0
+# Try not to wake up too often
+POLL_TIME_BLOCKING = 300.0
```

Comment:
At 300s block time:

~ 5 idle thread wakes per minute with 25 devices

Reply:
I don't think there should be any timeout here at all, we have a socketpair with the purpose of waking up the thread in a timely manner and we should have removed the timeout when we added the socketpair.